pkg/engine/lifecycletest/pending_replace_test.go

```diff
@@ -349,6 +349,9 @@ func TestPendingReplaceResumeWithSameGoals(t *testing.T) {
 	removeHostF := deploytest.NewPluginHostF(nil, nil, programF, removeLoaders...)
 	removeOptions := lt.TestUpdateOptions{T: t, HostF: removeHostF}
 
+	removeOptions.Refresh = true
```

Comment:
Feel like it would be good to run this test twice, once with these set and once continuing to test without these set.